assets/src/dashboard/app/views/exploreTemplates/header/index.js

```diff
@@ -22,102 +22,38 @@ import { __ } from '@web-stories-wp/i18n';
 /**
  * External dependencies
  */
-import styled from 'styled-components';
 import PropTypes from 'prop-types';
 import { useFeature } from 'flagged';
 
 /**
  * Internal dependencies
  */
-import { Dropdown } from '../../../../components';
-import { DropdownContainer } from '../../../../components/dropdown';
 import {
   DASHBOARD_VIEWS,
   TEMPLATES_GALLERY_SORT_MENU_ITEMS,
-  DROPDOWN_TYPES,
 } from '../../../../constants';
 import {
   FilterPropTypes,
-  SearchPropTypes,
   SortPropTypes,
   ViewPropTypes,
 } from '../../../../utils/useTemplateView';
 import { useDashboardResultsLabel } from '../../../../utils';
-import { TemplatesPropType } from '../../../../types';
 import { PageHeading, BodyViewOptions } from '../../shared';
-import useTemplateFilters from '../templateFilters';
 
-const HeadingDropdownsContainer = styled.div`
-  display: flex;
-  align-items: baseline;
-  flex-direction: row;
-  justify-content: center;
-  width: 100%;
-
-  ${DropdownContainer} {
-    margin: 0 10px;
-    &:last-child {
-      margin-right: 0;
-    }
-  }
-`;
-function Header({ filter, totalTemplates, search, templates, sort, view }) {
+function Header({ filter, totalTemplates, sort, view }) {
   const enableInProgressTemplateActions = useFeature(
     'enableInProgressTemplateActions'
   );
 
   const resultsLabel = useDashboardResultsLabel({
-    isActiveSearch: Boolean(search.keyword),
     totalResults: totalTemplates,
     currentFilter: filter.value,
     view: DASHBOARD_VIEWS.TEMPLATES_GALLERY,
   });
 
-  const {
-    selectedCategories,
-    selectedColors,
-    onNewCategorySelected,
-    onNewColorSelected,
-    clearAllCategories,
-    clearAllColors,
-  } = useTemplateFilters();
-
-  const TemplateFilters = enableInProgressTemplateActions ? (
-    <HeadingDropdownsContainer>
-      <Dropdown
-        ariaLabel={__('Category Dropdown', 'web-stories')}
-        type={DROPDOWN_TYPES.PANEL}
-        placeholder={__('Category', 'web-stories')}
-        items={selectedCategories}
-        onClear={clearAllCategories}
-        onChange={onNewCategorySelected}
-      />
-      <Dropdown
-        ariaLabel={__('Color Dropdown', 'web-stories')}
-        type={DROPDOWN_TYPES.COLOR_PANEL}
-        placeholder={__('Color', 'web-stories')}
-        items={selectedColors}
-        onClear={clearAllColors}
-        onChange={onNewColorSelected}
-      />
-    </HeadingDropdownsContainer>
-  ) : null;
-
```

Comment:
This is all tragically out of date. When it comes time to actually wire up filters I can't see this being that useful to have, it'll have to be redone. We need an API for the templates before this can happen at all. Taking it out to a) remove dead code and b) prevent further situations like the one last week where the in progress templates flag was turned on in staging and confused some newer team members :)